content/docs/other-guides/support.md

```diff
@@ -0,0 +1,201 @@
+status. To see the status of each application, refer to the 
+[Kubeflow application 
+matrix](/docs/reference/version-policy/#kubeflow-application-matrix) on the
+version policies page.
+
+The following table describes the level of support that you can expect based on
+the status of an application:
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Application status</th>
+        <th>Level of support</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Stable</td>
+        <td>The Kubeflow community offers <i>best-effort support</i> for stable
+          applications. See the section on 
+          <a href="#community-support">community support</a> below for a
+          definition of best-effort support and the community channels where you 
+          can report and discuss the problem. You can also consider requesting 
+          support from a 
+          <a href="#provider-support">Kubeflow community provider</a> or from 
+          your <a href="#cloud-support">cloud provider</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>Beta</td>
+        <td>The Kubeflow community offers <i>best-effort support</i> for beta
+          applications. See the section on 
+          <a href="#community-support">community support</a> below for a
+          definition of best-effort support and the community channels where you 
+          can report and discuss the problem. 
+        </td>
+      </tr>
+      <tr>
+        <td>Alpha</td>
+        <td>The response differs per application in alpha status, depending on
+          the size of the community for that application and the current level
+          of active development of the application.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="community-support">
+## Support from the Kubeflow community
+
+Kubeflow has an active and helpful community of users and contributors. 
+
+The Kubeflow community offers support on a best-effort basis for stable and beta
+applications.
+**Best-effort support** means that there's no formal agreement or
+commitment to solve a problem but the community appreciates the
+importance of addressing the problem as soon as possible. The community commits
+to helping you diagnose and address the problem if all the following are true:
+
+* The cause falls within the technical framework that Kubeflow controls. For
+  example, the Kubeflow community may not be able to help if the problem is 
+  caused by a specific network configuration within your organization.
+* Community members can reproduce the problem.
+* The reporter of the problem can help with further diagnosis and 
+  troubleshooting.
+
+You can ask questions and make suggestions in the following places:
+
+* **Slack** for online chat and messaging. See details of Kubeflow's 
+  [Slack workspace and channels](/docs/about/community/#slack).
```

Comment:
This link should work on the current live site, but didn't work in deploy-preview--it went to the correct page, but didn't jump down to the Slack area (probably because it's been edited and isn't primarily a Slack section; it's labeled **Forums and mailing groups**).

Reply:
Thanks for pointing this out. The reason is that I added the section to the community page after creating this PR, specifically so that I can link to it from this PR. The link will (should) work in prod, as you say.